ManatiCR/retopais
# Add humpback and move to CircleCI.

How to test:
- composer install ; npm install
- ahoy up
- ahoy site local-settings
- ./node_modules/.bin/aquifer build
- ahoy site install-from-db

Site should work

Also, see deployment-multidev works: http://addhumpb-retopais.pantheonsite.io/ 

Credentials: incae/incae2018 :sweat_smile:  (I already changed them in circle config)
